sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -377,8 +394,9 @@ def _from_normal_time(self, value):
 
   def encode_to_stream(self, value, out, nested):
     span_micros = value.end.micros - value.start.micros
-    out.write_bigendian_uint64(self._from_normal_time(value.end.micros / 1000))
-    out.write_var_int64(span_micros / 1000)
+    out.write_bigendian_uint64(self._from_normal_time(
+        old_div(value.end.micros, 1000)))
```

Comment:
Prefer `//` to `old_div` if the types are integral (as they are here), and plain old `/` for floats.